service/src/test/java/bio/terra/workspace/service/workspace/ApplicationUnitTest.java

```diff
@@ -31,8 +30,8 @@
   // These strings have to match the text in WsmApplicationService or the test will fail
   private static final String ERROR_MISSING_REQUIRED =
       "Invalid application configuration: missing some required fields (identifier, service-account, state)";
-  private static final String ERROR_BAD_UUID =
-      "Invalid application configuration: invalid UUID format";
+  private static final String ERROR_BAD_ID =
```

Comment:
BTW, is it safe to have an application ID start with a digit? If so, that limits the contexts where it can be used (in scripts, etc).

Reply:
Yes, it can start with a number. The pattern is `"^[a-zA-Z0-9][-_a-zA-Z0-9]{0,1023}$"`

I don't think we need the application id to be useful as a shell variable name, so I think that is fine. Many WSM, GCP and Azure identifiers allow leading numbers.